### PR TITLE
Feature/1202246697694649 core send and receive charge point status events in protobuf format

### PR DESCRIPTION
**Pull request by irediaes**
added charge point gateway BootNotification, Heartbeat  and Log event proto messages definition

### CODE REVIEW

File: charge_point_gateway_format.proto [hunk 1]
```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package gateway;
+option go_package = "github.com/peaqnetwork/peaq-network-ev-charging-message-format/golang/gateway;gateway";
+
+enum EventType {
+  BootNotificationRequest = 0;
```
Comment (sfffaaa) [inline charge_point_gateway_format.proto:7]: Is there a special reason why the EnumType isn't continual?

Comment (irediaes) [reply charge_point_gateway_format.proto:7]: > ...the EnumType isn't continual

Please explain..I don't understand this

Comment (sfffaaa) [reply charge_point_gateway_format.proto:7]: I assume the data in Enum will be continuous normally. For example,
```
enum EventType {
  BootNotificationRequest = 0;
  HeartbeatRequest = 1;
  Log = 2;
}
```
That's why I want to ask whether we have a special reason why we set up the `HeartbeatRequest = 3` but not `HeartbeatRequest = 1`.

Comment (irediaes) [reply charge_point_gateway_format.proto:7]: Ooops...yes it was an oversight. The enum had fields which I deleted but forgot to update the numbering.

Comment (irediaes) [reply charge_point_gateway_format.proto:7]: Fixed

**Review by sfffaaa — APPROVED** [review]

**Pull request merged**